quiffen/core/qif.py

```diff
@@ -109,6 +109,7 @@ def parse(
         separator: str = "\n",
         day_first: bool = False,
         encoding: str = "utf-8",
+        locale: str = 'en_US'
```

Comment:
Given the limited number of locales available, would it not be better to use an enum? Perhaps we could have something like:

```python
class DecimalSeparator(enum.Enum):
    PERIOD = "."
    COMMA = ","
```